include/aspect/introspection.h

```diff
@@ -268,9 +268,21 @@ namespace aspect
 
         /**
          * Pressure unknowns that are locally owned. This IndexSet is needed
-         * if velocity and pressure end up in the same block.
+         * if velocity and pressure end up in the same block. If melt transport
+         * is enabled, this will contain both pressures.
          */
         IndexSet locally_owned_pressure_dofs;
+
+        /**
+         * Fluid and compaction pressure unknowns that are locally owned. Only valid if
+         * melt transport is enabled.
+         */
+        IndexSet locally_owned_melt_pressure_dofs;
```

Comment:
From the description I do not see the difference to the ´locally_owned_pressure_dofs´ above. The description of that one states it contains both pressures, and I thought you had only two pressures?